teps/0079-tekton-catalog-support-tiers.md

```diff
@@ -100,14 +122,283 @@ requirements, and provenance attestations can be made to meet software supply ch
 [TEP-0091: Verified Remote Resources][tep-0091] will flesh out the details of signing, while this TEP will focus on
 surfacing the verification information and building a corpus of verified resources that users can trust.
 
+## Proposal
+
+### Ownership and Maintenance
+
+As previously discussed in [TEP-0003][tep-0003-upstream], we propose creating 
+two support tiers, `Community` and `Official`, through separate Tekton Catalogs.
+
+A Catalog is a git repository that complies with the organization contract 
+defined in [TEP-0003: Tekton Catalog Organization][tep-0003-org]. 
+
+The Community Catalog makes it easy for contributors to share resources, 
+while the Official Catalog provides a corpus of high quality resources 
+that users can rely on.
+
+Both the Community and Official Catalogs will be:
+* Owned by the [Tekton Catalog Maintainers][catalog-owners].
```

Comment:
are the contents of the community catalog also owned by the catalog maintainers? i would have thought the individual items in the community catalog would be owned by the folks who contributed them

Reply:
the community catalog itself is owned by the catalog maintainers at the top level so that they can maintain the general infrastructure, but the resources in the community catalog are owned by the contributors - added clarification on this